packages/virtualized-list/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@solid-primitives/virtualized-list",
+  "version": "0.0.100",
+  "description": "A virtualized list component for performantly rendering lists with many elements",
+  "author": "Spencer Whitehead <spencerwhitehead7@gmail.com>",
+  "contributors": [],
+  "license": "MIT",
+  "homepage": "https://primitives.solidjs.community/package/virtualized-list",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/solidjs-community/solid-primitives.git"
+  },
+  "bugs": {
+    "url": "https://github.com/solidjs-community/solid-primitives/issues"
+  },
+  "primitive": {
+    "name": "virtualized-list",
+    "stage": 0,
+    "list": [
+      "VirtualizedList"
```

Comment:
Also add VirtualListItem so the size of the primitive will be shown correctly.

Reply:
I'm not sure what this means; the VirtualListItem comps are just for the demo/unit tests. They weren't intended to be part of the package itself.

Reply:
If that is the case, please do not export them from the package, but instead move them to the demo.